### ISSUE TITLE
idiopathic recurrent rhabdomyolysis

### ISSUE THREAD

**Issue by Daniel-Olson** [obophenotype/human-phenotype-ontology#11489]
**Preferred term label:**
idiopathic recurrent rhabdomyolysis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Recurrent episodes of rhabdomyolysis, characterized by repeated acute breakdown of skeletal muscle with marked elevation of serum creatine kinase and often myoglobinuria, occurring in the absence of an identifiable precipitating cause or underlying disorder.

**Parent term (use hpo.jax.org/app)**
Rhabdomyolysis HP:0003201

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-8134-1207

**Comment by pnrobinson** [comment]
@Daniel-Olson 
From what I can see this term is referring to a disease characterized by recurrent rhabdo, myoglobinuria, pain, weakness with unknown cause -- 
Most of the citations about this are rather old, e.g.: https://pubmed.ncbi.nlm.nih.gov/13976533/
In the HPO system we would count this as a disease of unknown cause characterized by recurrent rhabdo
Perhaps there is a case to be made for adding a new term "Recurrent rhabdomylosis" (without the "idiopathic", which refers to a disease!)?

**Comment by pnrobinson** [comment]
@Daniel-Olson  -- will close for now, get in touch if this did not solve your issue!
